Jenkinsfile

```diff
@@ -38,6 +44,23 @@ pipeline {
             }
         }
 
+        stage('Prepare-Logs-Destination') {
+            agent { label 'master' }
```

Comment:
Is there any particular reason why this executed on master? Is it because env variables are only present there?

If that's the case, I would try to move this to `Preparation` stage somehow. If you think about it, `Preparation` stage, which basically stashes CI code, has no reason why it requires any ansible. Maybe it should be `master` there?